index.js

```diff
@@ -242,27 +127,23 @@ function isLoggedIn(req) {
 var port = config.port;
 var httpServer = require('http').createServer(app);
 httpServer.listen(port, function() {
-    console.log('Utopian 1UP running on port ' + port + '.');
+    console.log('1UP running on port ' + port + '.');
 });
 
 // This will enable the Live Query real-time server
-ParseServer.createLiveQueryServer(httpServer);
+// ParseServer.createLiveQueryServer(httpServer);
```

Comment:
Commented out code is not benefitial, and because the code is on github, where everybode can browse the history, it's not needed.